internal/internal_test.go

```diff
@@ -380,6 +428,67 @@ func TestCheckDenyObjectDecision(t *testing.T) {
 	}
 }
 
+func TestCheckDenyWithDryRunObjectDecision(t *testing.T) {
+
+	var req ext_authz.CheckRequest
+	if err := util.Unmarshal([]byte(exampleDeniedRequest), &req); err != nil {
+		panic(err)
+	}
+
+	server := testAuthzServerWithObjectDecision(&testPlugin{}, true)
+	ctx := context.Background()
+	output, err := server.Check(ctx, &req)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	fmt.Printf("Result is %v\n", output)
+
+	if output.Status.Code != int32(google_rpc.OK) {
+		t.Fatalf("Expected request to be allowed since config.DryRun is true, but got: %v", output)
+	}
+
+	response := output.GetOkResponse()
+	if response == nil {
+		t.Fatal("Expected OkHttpResponse struct but got nil")
+	}
```

Comment:
Related to comment about empty `HttpResponse `.

Reply:
See my other comment. lmk what you think.